src/test/java/jp/co/soramitsu/sora/didresolver/controllers/DIDResolverControllerTest.java

```diff
@@ -1,18 +1,23 @@
 package jp.co.soramitsu.sora.didresolver.controllers;
 
+import static java.util.Optional.of;
+import static jp.co.soramitsu.sora.didresolver.commons.URIConstants.ID_PARAM;
+import static jp.co.soramitsu.sora.didresolver.commons.URIConstants.PATH;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.when;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 import java.time.Instant;
-import java.util.Optional;
 import jp.co.soramitsu.sora.didresolver.exceptions.DIDNotFoundException;
+import jp.co.soramitsu.sora.sdk.did.model.dto.DID;
+import jp.co.soramitsu.sora.sdk.did.parser.generated.ParserException;
 import org.junit.Test;
```

Comment:
You are removed dependency to junit vintage engine, but didn't change tests to junit 5. How it works right now?

Reply:
junit4 derives from springboot-starter-test and Test class does not use `Runner` of junit5